Client/mods/deathmatch/logic/CClientMarker.cpp

```diff
@@ -89,6 +89,18 @@ void CClientMarker::SetPosition(const CVector& vecPosition)
     UpdateStreamPosition(vecPosition);
 }
 
+void CClientMarker::AttachTo(CClientEntity* pEntity)
+{
+    CClientEntity::AttachTo(pEntity);
+    if (m_pAttachedToEntity)
+    {
+        CVector pVector;
+
+        pEntity->GetPosition(pVector);
+        SetPosition(pVector + m_vecAttachedPosition);
+    }
```

Comment:
```suggestion
    m_pCollision->AttachTo(pEntity);
```
Have you considered just calling `m_pCollision->AttachTo(pEntity)`?